src/Component/spec/FieldTypes/EnumFieldTypeSpec.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Component\Grid\FieldTypes;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Component\Grid\DataExtractor\DataExtractorInterface;
+use Sylius\Component\Grid\Definition\Field;
+use Sylius\Component\Grid\FieldTypes\FieldTypeInterface;
+use Sylius\Component\Grid\Tests\Dummy\TestEnum;
+use Sylius\Component\Grid\Tests\Dummy\TestNonBackedEnum;
+use Sylius\Component\Grid\Tests\Dummy\TestTranslatableEnum;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+final class EnumFieldTypeSpec extends ObjectBehavior
```

Comment:
Please, create a PHPUnit test instead.
We are currently removing all Phpspec tests.